lib/scraping/hpi_table_scraper.rb

```diff
@@ -0,0 +1,33 @@
+require "#{Rails.root}/lib/scraping/hpi_web_scraper.rb"
+
+class HpiTableScraper < HpiWebScraper
+  def scrape
+    item = {}
+
+    item[:email] = @html.css('.mail').text
+
+    td_tags = @html.css('td')
+    td_tags.each do |td|
+      # Phone
+      if ([td.text] & @@phone_words).any? # Check if td_text is one of those words
+        index = td_tags.find_index(td) # Save the index of the td in the td_tags array
+        item[:phone] = td_tags[index + 1].text # Next td in the array has to contain the phone number
+      end
+      # Office
+      if ([td.text] & @@office_words).any?
+        index = td_tags.find_index(td)
+        item[:office] = td_tags[index + 1].text
+        item[:office] = td_tags[index + 2].text if item[:office] == '' # In case of separator
+      end
+      # Email
+      next unless item[:email] == ''
```

Comment:
Can we move line 7 down here and do a simple if-elseif (with if-block at line 25) or do we have to split it up like this?

Then we could get rid of this `next`